mix.exs

```diff
@@ -7,7 +7,7 @@ defmodule Jalaali.Mixfile do
     [
       app: :jalaali,
       version: @version,
-      elixir: "~> 1.3",
+      elixir: "~> 1.12.0",
```

Comment:
Please revert this. 

We should use something like `"~> 1.6"` as if you use `"~> 1.12"` I think we will loose backwards compatibility with older versions of elixir. Generally only increase this if you use a functionality which is only existing in the newer version. But this needs to be done in another PR with and investigated on its own.

Reply:
Unfortunately,  I have no older elixir version to test it, but as I saw nothing was deprecated I think it is compatible with elixir 1.6

I will try to update full version of this lib for elixir 1.12, it should be noted we have `22 doctests, 13 tests, 4 failures`